src/OrleansSQLUtils/CreateOrleansTables_MySql.sql

```diff
@@ -248,14 +328,156 @@ BEGIN
 	SET _ROWCOUNT = ROW_COUNT();
 
 	IF _ROWCOUNT = 0
-	THEN 
+	THEN
 		ROLLBACK;
 	ELSE
 		COMMIT;
 	END IF;
 	SELECT _ROWCOUNT;
 END$$
 
+CREATE PROCEDURE ClearStorage
```

Comment:
Both `ClearStorage` and `WriteToStorage` can be implemented without `IF` statements or explicit transactions, see `UpsertReminderRowKey`. I suggest the following:
1) Keep the `WriteToStorage`, simplify it according to `UpsertReminderRowKey`
2) Remove the `ClearStorage` procedure
3) The `ClearStorageKey` should call `WriteToStorage` with null payload. 
This removes the existing code duplication. The same goes for the MSSQL script.

As a side note, when I look at it now it seems it'd be better if we keep the tables creation and related queries in "regions". A membership section, a reminders section, etc. It isn't directly related to this PR but the script is getting big, and that might help organize it, what do you think ? If you're OK with it, I'll do it